apps/pagetour-sdk/src/common/configstore.ts

```diff
@@ -67,13 +67,34 @@ class ConfigStore {
     },
     zIndex: "100000",
     textAreaCharacterLength: 500,
-    enableBeacon: false
+    enableBeacon: false,
+    feedback : {
+      pagetourFeedbackOptions: {
+        enabled: false,
+        type:'5-star',
+        heading: "Feedback : PageTour",
+        description: "Was this tour helpful?",
```

Comment:
Same here

Reply:
Changed to camel casing - PagetourFeedbackOptions